backend/src/test/java/com/htecgroup/skynest/utils/UserDtoUtil.java

```diff
@@ -28,4 +30,8 @@ public static UserDto getNotVerified() {
   public static UserDto getVerified() {
     return getNotVerified().withVerified(true).withEnabled(true);
   }
+
+  public static UserDto getNotWorker() {
```

Comment:
why not simple call this method to createManager or something that can be reusable in some other situation ?